stapi-fastapi/src/stapi_fastapi/routers/product_router.py

```diff
@@ -81,6 +83,16 @@ def __init__(
                 f"Product '{product.id}' must support async opportunity search since the root router does",
             )
 
+        product_conformances = PRODUCT_CONFORMANCE.all()
+        has_geosjon = False
+        for conformance in product.conformsTo:
+            if conformance not in product_conformances:
+                raise ValueError(f"{conformance} is not a valid product conformance")
```

Comment:
Is there a reason to prevent advertising conformance classes that are not part of the spec? This seems like it could prevent easily developing/implementing new extensions or the like. Maybe a warning would be better than an exception?

Reply:
Nope, I agree that we should be more flexible. I'll remove this check.